data/rankdeficient_test.csv

```diff
@@ -0,0 +1,1001 @@
+x1,x2,y
```

Comment:
Better to simulate a dataset instead of checking in datasets.

Reply:
Sorry, I do not quite understand... What do you mean by "simulate a dataset"? Generate a random dataset every time when running the test code?

Reply:
yes, but if you set the random seed it will be the same every time.